vmck/api.py

```diff
@@ -1,13 +1,21 @@
 import json
+import logging
+
 from django.conf import settings
 from django.http import JsonResponse
 from django.urls import path
 from django.views.decorators.http import require_http_methods
 from django.views.decorators.csrf import csrf_exempt
 from django.shortcuts import get_object_or_404
-from .backends import get_backend
-from . import jobs
-from . import models
+
+from vmck.backends import get_backend
+from vmck import jobs
+from vmck import models
+
+
+log_level = logging.WARNING
```

Comment:
You can use a more permissive log level, info or debug, it will get filtered by the log handler later.